src/commands/android/subcommands/index.ts

```diff
@@ -0,0 +1,65 @@
+import path from 'path';
+import colors from 'ansi-colors';
+import * as dotenv from 'dotenv';
+
+import {connect} from './connect';
+import Logger from '../../../logger';
+import {getPlatformName} from '../../../utils';
+import {Options, Platform} from '../interfaces';
+import {checkJavaInstallation, getSdkRootFromEnv} from '../utils/common';
+
+export class AndroidSubcommand {
+  sdkRoot: string;
+  options: Options;
+  subcommand: string;
+  rootDir: string;
+  platform: Platform;
+  androidHomeInGlobalEnv: boolean;
+
+  constructor(subcommand: string, options: Options, rootDir = process.cwd()) {
+    this.sdkRoot = '';
+    this.options = options;
+    this.subcommand = subcommand;
+    this.rootDir = rootDir;
+    this.platform = getPlatformName();
+    this.androidHomeInGlobalEnv = false;
+  }
+
+  async run(): Promise<boolean> {
+    this.loadEnvFromDotEnv();
+
+    const javaInstalled = checkJavaInstallation(this.rootDir);
+    if (!javaInstalled) {
+      return false;
+    }
+
+    const sdkRootEnv = getSdkRootFromEnv(this.androidHomeInGlobalEnv, this.rootDir);
+    if (!sdkRootEnv) {
+      Logger.log(colors.red('Path to Android SDK not found in environment variables!\n'));
+      Logger.log(`Run: ${colors.cyan('npx @nightwatch/mobile-helper android --standalone')} to setup Android SDK`);
+      Logger.log('     or provide the path to Android SDK if already installed.');
```

Comment:
why this line? where would they provide the path to Android SDK? I think running the setup command would already handle this and they would need to provide the Android SDK path there.